### PR TITLE
Join the hotline webring

### PR DESCRIPTION
**Pull request by gabebw**
This can only mean one thing!

Docs, of a sort: http://hotlinewebring.club/

### CODE REVIEW

File: source/layouts/header.html.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %h1.blog-title
-  = link_to('←', 'http://alexfox.me', title: 'Yes, this is a webring')
+  = link_to('←', 'http://hotlinewebring.club/adarsh/previous', title: 'Yes, this is a webring')
   = link_to('Adarsh Pandit', '/')
-  = link_to('→', 'http://gabebw.com', title: 'Yes, this is a webring')
+  = link_to('→', 'http://hotlinewebring.club/adarsh/next', title: 'Yes, this is a webring')
```
Comment (gabebw) [inline source/layouts/header.html.haml]: How should I format this? Haml doesn't allow:

```haml
= link_to(
  '←',
    'http://hotlinewebring.club/adarsh/previous',
    title: 'Yes, this is a webring'
  )
```

Comment (adarsh) [inline source/layouts/header.html.haml]: Forget it, we have bigger fish to fry.

**Comment by adarsh** [comment]
nice shot

**Comment by gabebw** [comment]
WOO HOO

**Pull request merged**